### PR TITLE
Combine Windows install pages

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Combined Windows Home and Windows Install pages .

### CODE REVIEW

**Comment by usha-mandya** [comment]
@StefanScherer PTAL

File: docker-for-windows/install.md [hunk 1]
```diff
@@ -32,6 +32,23 @@ Hyper-V on Windows 10:
     BIOS settings.  For more information, see
     [Virtualization](troubleshoot.md#virtualization-must-be-enabled).
 
+### Windows 10 Home system requirements
+
+ Windows 10 Home machines must meet the following requirements to install Docker Desktop:
+
+  - Install Windows 10, version 1903 or higher.
```
Comment (StefanScherer) [inline docker-for-windows/install.md:39]:
```suggestion
  - Windows 10 64-bit: Home, Pro, Enterprise, or Education, version 1903 (Build 18362) or higher.
``

File: docker-for-windows/install.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ By downloading Docker Desktop, you agree to the terms of the [Docker Software En
 
 ## What to know before you install
```
Comment (StefanScherer) [inline docker-for-windows/install.md:18]: Maybe remove this headline to be in sync with the Mac installation page?

```suggestion
## System requirements
```

File: docker-for-windows/install.md [hunk 3]
```diff
@@ -17,11 +17,11 @@ By downloading Docker Desktop, you agree to the terms of the [Docker Software En
...
 
-### System Requirements
+### System requirements
```
Comment (StefanScherer) [inline docker-for-windows/install.md:20]:
```suggestion
### System requirements for Hyper-V backend and/or Windows containers
```

File: docker-for-windows/install.md [hunk 4]
```diff
@@ -32,6 +32,23 @@ Hyper-V on Windows 10:
     BIOS settings.  For more information, see
     [Virtualization](troubleshoot.md#virtualization-must-be-enabled).
 
+### Windows 10 Home system requirements
```
Comment (StefanScherer) [inline docker-for-windows/install.md:35]:
```suggestion
### System requirements for WSL 2 backend
```

File: docker-for-windows/install.md [hunk 5]
```diff
@@ -32,6 +32,23 @@ Hyper-V on Windows 10:
...
+
+ Windows 10 Home machines must meet the following requirements to install Docker Desktop:
```
Comment (StefanScherer) [inline docker-for-windows/install.md:37]: Maybe an introcution like:

Windows 10, version 1903 or higher is capable of running WSL 2. We recommend using the Docker Desktop WSL 2 backend when possible.

File: docker-for-windows/install.md [hunk 6]
```diff
@@ -32,6 +32,23 @@ Hyper-V on Windows 10:
...
+
+  - Install Windows 10, version 1903 or higher.
+  - Enable the WSL 2 feature on Windows. For detailed instructions, refer to the
+    [Microsoft documentation](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+  - The following hardware prerequisites are required to successfully run
+WSL 2 on Windows 10 Home:
```
Comment (StefanScherer) [inline docker-for-windows/install.md:43]:
```suggestion
WSL 2 on Windows 10:
```

File: docker-for-windows/install.md [hunk 7]
```diff
@@ -17,11 +17,11 @@ By downloading Docker Desktop, you agree to the terms of the [Docker Software En
...
 ## What to know before you install
 
-### System Requirements
+### System requirements
 
   - Windows 10 64-bit: Pro, Enterprise, or Education (Build 17134 or later).
-  
-    For Windows 10 Home, see [Install Docker Desktop on Windows Home](install-windows-home.md).
+
+    For Windows 10 Home, see [Windows 10 Home system requirements](#windows-10-home-system-requirements).
```
Comment (StefanScherer) [inline docker-for-windows/install.md:24]:
```suggestion
    For Windows 10 Home, see [System requirements for WSL 2 backend](#system-requirements-for-wsl-2-backend).
```

**Review by StefanScherer — COMMENTED** [review]
Good idea.
I have some suggestions that I tried to put into review blocks, but that maybe a bit difficult to read instead of a finished netlify preview page 😅 
WDYT?

**Comment by usha-mandya** [comment]
Thanks @StefanScherer. Might be easier with Netlify preview. I've added the suggestions. Will take a look at the updated version.

File: docker-for-windows/install.md (hunk 5 above)
Comment (StefanScherer) [reply docker-for-windows/install.md:37]: Looks very good, we now can remove this line and I think we have everything covered between Pro and Home.

Comment (usha-mandya) [reply docker-for-windows/install.md:37]: Yea, I'm wondering if we should also remove the repetition 'system requirements' in the subsection

Comment (StefanScherer) [reply docker-for-windows/install.md]: That's a great idea!

**Review by StefanScherer — APPROVED** [review]
LGTM 👌🎉

**Pull request merged**